CoffeeShop/src/main/java/interfaces/EntityList.java

```diff
@@ -0,0 +1,15 @@
+package interfaces;
+
+import exceptions.DuplicateItemIDException;
+
+/**
+ * @author Fraser Holman
+ *
+ * Interface used for the Order and Item List Classes
+ */
+
+public interface EntityList<T, R> {
+    void add(T item) throws DuplicateItemIDException;
+
+    void remove(R ID);
```

Comment:
```suggestion
    Boolean remove(R ID);
```

this goes together with the OrderList remove comment